api/src/main/webapp/WEB-INF/app.yaml.template

```diff
@@ -0,0 +1,30 @@
+service: api
+runtime: java11
+threadsafe: true
+application: "all-of-us-workbench-test"
+env_variables:
+  GAE_PROFILER_MODE: "cpu,heap"
+  JAVA_USER_OPTS: -Djava.util.logging.config.file=webapps/root/WEB-INF/logging.properties
+  SPRING_DATASOURCE_DRIVER_CLASS_NAME=${DB_CONNECTION_STRING}
+  SPRING_DATASOURCE_URL=${DB_CONNECTION_STRING}
+  SPRING_DATASOURCE_USERNAME=${WORKBENCH_DB_USER}
+  SPRING_DATASOURCE_PASSWORD=${WORKBENCH_DB_PASSWORD}
+  CDR_DATASOURCE_URL=${CDR_DB_CONNECTION_STRING}
+  CDR_DATASOURCE_USERNAME=${CDR_DB_USER}
+  CDR_DATASOURCE_PASSWORD=${CDR_DB_PASSWORD}
+error_handlers:
+  - file: server_unavailable.html
+handlers:
+-url: /static/**
+  static_dir: static
+  http_headers:
+    Strict-Transport-Security: "max-age=31536000; includeSubDomains; preload"
+    X-XSS-Protection: "1"
+    X-Content-Type-Options: "nosniff"
+    Content-Security-Policy: "default-src 'none'; frame-ancestors 'none'; report-uri /content-security-report"
+instance_class: "${GAE_INSTANCE_CLASS}"
+automatic_scaling:
+ min_idle_instances "${GAE_MIN_IDLE_INSTANCES}"
+ max_instances: "${GAE_MAX_INSTANCES}"
+inbound_services:
```

Comment:
seems we need this to use min_idle_instance: 
https://cloud.google.com/appengine/docs/standard/java11/configuring-warmup-requests
not sure why it works before...